### PR TITLE
fix low contrast

### PR DESCRIPTION
**Pull request by ZirixCZ**
Fixes https://github.com/Vanilla-OS/website/issues/87

### CODE REVIEW

**Comment by ZirixCZ** [comment]
Pre
obrazek
Post
obrazek

**Review by kbdharun — APPROVED** [review]

File: assets/css/dark.css [hunk 1]
```diff
@@ -12,7 +12,7 @@
     }
 
     .hero p {
-        color: #c3c3c3;
+        color: #848484;
```
Comment (TheEvilSkeleton) [inline assets/css/dark.css:15]: I feel like `#848484` is still too low (although, much better). Perhaps we could try something lighter?

**Review by TheEvilSkeleton — CHANGES_REQUESTED** [review]

**Comment by TheEvilSkeleton** [comment]
We should merge https://github.com/Vanilla-OS/website/pull/84 first, imo. We can pick a better color without dealing with inversion.

File: assets/css/dark.css (hunk 1 above)
Comment (GabsEdits) [reply assets/css/dark.css:15]: > I feel like `#848484` is still too low (although, much better). Perhaps we could try something lighter?

Yep, I agree, We should go with something above `AA` for contrast, I recommend `#A3A3A3`, it's almost `AAA` so, the contrast should be good for most users.

**Reference:** [Use Contrast](https://usecontrast.com/guide)
**Contrast Ratio for `#A3A3A3`:** 6.2:1
**How the text looks:** 


```suggestion
        color: #A3A3A3;
```

Comment (ZirixCZ) [reply assets/css/dark.css:15]: Even though it looks visually darker, `#A3A3A3` has better contrast than `#848484`
LGTM

`#A3A3A3`
Screenshot 2023-06-10 at 15 40 47

`#848484`
Screenshot 2023-06-10 at 15 41 19

**Review by GabsEdits — APPROVED** [review]

**Comment by TheEvilSkeleton** [comment]
I apologize for making you wait, but we've merged https://github.com/Vanilla-OS/website/pull/92, which addresses this MR. We're closing this.

**Pull request closed without merging**